articles/users/normalized/auth0/normalized-user-profile-schema.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Normalized User Profile schema reference. 
+topics:
+  - user-profile
+  - normalized-user-profile
+  - users
+  - auth0-user-profiles
+contentType:
+  - reference
+useCase:
+  - manage-users
+v2: true
+---
+# Normalized User Profile Schema
+
+The attributes that Auth0 maps to a common schema are listed below.
+
+Fields that are always generated:
+
+* **`name`**: the user's full name.
+* **`nickname`**: the user's username.
+* **`picture`**: the URL of the [user's picture](/users/guides/change-user-pictures). If unavailable, Auth0 uses the Gravatar image associated with the user's email address.
+* **`user_id`**: the user's unique identifier. This is unique per Connection, but the same for all apps that authenticate via that Connection.
```

Comment:
This line isn't clear to me

Reply:
Might need further information after technical review. Not changed for this PR; just chunked as it was.